portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,21 +31,44 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
-
-    ArrayList<String> messages = new ArrayList<>(Arrays.asList("Hello", "Hola", "Bonjour"));
-
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Do you know if acquiring the datastore service is resource-intensive? Is this generally done per-request, or during an initialization call?

Reply:
I didn't. That's good to know. I added an initialization call to my servlets